packages/component/src/BasicWebChat.js

```diff
@@ -41,18 +42,28 @@ const TRANSCRIPT_CSS = css({
   flex: 1
 });
 
+const SilentError = ({ err, message }) => {
```

Comment:
Would have preferred to have this as a hook but it looks like you are using in the `activityMiddleware`.... sad.

Reply:
😅 Exactly! This one is ugly.

We need to converge `<ErrorBox>`, `<ErrorBoundary>`, and probably `<SilentError>` into a single easy-to-use component.